server/src/github.com/alexeyknyshev/tools/metro_json_to_towns_db/parser.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	_ "github.com/mattn/go-sqlite3"
+	"log"
+	"os"
+	"strconv"
+)
+
+func dbOpen(path string) (*sql.DB, error) {
+	db, err := sql.Open("sqlite3", path)
+	if err != nil {
+		return nil, err
+	}
+	_, err = db.Exec(`CREATE TABLE metro (
+						id INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
CREATE TABLE IF NOT EXISTS ...
DELETE FROM metro -- clear table
DELETE FROM SQLITE_SEQUENCE WHERE name='metro' -- reset index autoincrement counter